pkg/api/dtos/prefs.go

```diff
@@ -26,8 +25,8 @@ type UpdatePrefsCmd struct {
 	// Enum: utc,browser
 	Timezone     string                         `json:"timezone"`
 	WeekStart    string                         `json:"weekStart"`
-	Navbar       *models.NavbarPreference       `json:"navbar,omitempty"`
-	QueryHistory *models.QueryHistoryPreference `json:"queryHistory,omitempty"`
+	Navbar       *pref.NavbarPreference         `json:"navbar,omitempty"`
+	QueryHistory *pref.QueryHistoryPreference   `json:"queryHistory,omitempty"`
```

Comment:
I would suggest getting rid of `models.NavbarPreference` and `models.QueryHistoryPreference` and use everywhere the models defined under the `preference` package, if possible

Reply:
I got rid of them in https://github.com/grafana/grafana/pull/49677/commits/6f7aa9f8d35a7aad505fa3bad7e2ddeaddb4c527

Reply:
it seems that it's not possible after all due to cycle dependencies; sorry!